Applications/DataExplorer/DataView/GEOModels.h

```diff
@@ -76,6 +76,9 @@ public slots:
     void appendSurfaceVec(std::string const& name);
     void removeSurfaceVec(std::string const& name);
 
+    void renameGeometry(std::string const& old_name,
```

Comment:
No Documentation, but maybe thats not mandatory here...

Reply:
@endJunction: Almost none of the methods in this class are documented. So personally I'd rather not be too picky here.

Reply:
Yes, not here. That's fine. ;)
Tom added the docu at the right place, where most functions are documented. :+1: 
:stop_button: